scripts/test

```diff
@@ -4,7 +4,8 @@ set -e
 projectRoot="$(a="/$0"; a=${a%/*}; a=${a:-.}; a=${a#/}/; cd "$a/.." || return; pwd)"
 
 # Set Docker/Kubernetes resource names managed by this script
-containerName="ffc-demo-api-gateway-test"
+containerName="ffc-demo-api-gateway"
+projectName="ffc-demo-api-gateway-test"
```

Comment:
Mark's approach to this is to replace the containerName with serviceName, and us that as the -p setting (adding -test to the end) and also for the run command. This replaces two variables with just one, and looks slightly cleaner to me. Plus it should also work with the shortened service names that you were thinking about before.
https://github.com/DEFRA/ffc-demo-payment-service/pull/24/files

Reply:
Yep, we've now got both approaches already merged in different repos. Personally, I like the clarity of using two variables for two different purposes and think it reduces the likelihood of errors during future edits, since the `-test` is less noticeable in the middle of `docker-compose -p ${serviceName}-test -f docker-compose.yaml -f docker-compose.test.yaml $@`. I can also see the argument in favour of reducing duplication.

Either way works and will be equally usable with shortened service names.